src/sass/letters.scss

```diff
@@ -1,7 +1,13 @@
-@import "shared-variables";
-@import "modules/m-form-process";
-@import "modules/m-progress-bar";
-@import "modules/m-schemaform";
+@import 'shared-variables';
+@import 'modules/m-form-process';
+@import 'modules/m-progress-bar';
+@import 'modules/m-schemaform';
+@mixin schemaform-padding {
```

Comment:
Oh bother. We're not using schemaform in this app, so it probably doesn't make sense to import the css. I'll write up another ticket for that.

As far as this goes, it probably makes sense to rename this one to not include `schemaform-`.

Reply:
Good catch. We are using the `.schemaform-chapter-progress` class from `_m-schemaform.scss`, though. This is the wrapper that holds most of the app content, from the `StepHeader` on down. I named it this way to make it obvious that this style is basically there to align with what's defined in `_m-schemaform.scss`, but am definitely open to better suggestions.

Reply:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7150